### DISCUSSION TITLE
如何使用AMD显卡进行微调？

### DISCUSSION THREAD

**Discussion by purefire** [ymcui/Chinese-LLaMA-Alpaca-2#485] [Q&A]
看到requirement.txt，大概的想法是替换rocm版本的torch、支持rocm的transformers v4.36？其他代码中会有什么特化吗？

**Comment by iMountTai** [comment]
代码没有显式要求是cuda，可以试试

**Comment by purefire** [comment]
感谢，已经可用。安装rocm专用torch和bitsandbytes即可。
